tests.py

```diff
@@ -66,69 +66,73 @@ def testParseArguments(self):
     args = [prog, '-r', expected_docker_root, 'list', 'repositories']
     sys.argv = args
 
-    options = de_test_object.ParseArguments()
-    usage_string = de_test_object._argument_parser.format_usage()
+    options = de_object.ParseArguments()
+    usage_string = de_object._argument_parser.format_usage()
     expected_usage = '[-h] [-r DOCKER_DIRECTORY] {mount,list,history} ...\n'
     self.assertTrue(expected_usage in usage_string)
 
-    de_test_object.ParseOptions(options)
+    de_object.ParseOptions(options)
     self.assertEqual(expected_docker_root, options.docker_directory)
 
+  def testDetectStorageFail(self):
+    """Tests that the DockerExplorer.DetectStorage function fails."""
```

Comment:
As a general rule I try to make dosctrings in tests more specific. Here I would specify under which conditions the functinon should fail: `Tests that DockerExplorer.DetectStrorage fails when a nonexistent directory is specified`

Reply:
done